File: src/main/java/com/example/demo/dto/Wind.java
package com.example.demo.dto;

import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL)

public class Wind {
    String locationName;
    String direction;
    String speedMin;
    String speedMax;
    String gustSpeed;

    public Wind(String locationName, String direction, String speedMin, String speedMax, String gustSpeed) {
        this.locationName = locationName;
        this.direction = direction;
        this.speedMin = speedMin;
        this.speedMax = speedMax;
        this.gustSpeed = gustSpeed;
    }

    public String getLocationName() {
        return locationName;
    }

    public String getDirection() {
        return direction;
    }

    public String getSpeedMin() {
        return speedMin;
    }

    public String getSpeedMax() {
        return speedMax;
    }

    public String getGustSpeed() {
        return gustSpeed;
    }

}

File: app/src/app/datamodel/wind.ts
export class Wind {
  location: string
  direction: string
  speedMin: number
  speedMax: number
  gustSpeed: number

  constructor(location: string, direction: string, speedmin: number, speedmax: number, gustspeed:number) {
    this.location = location
    this.direction = direction
    this.speedMin = speedmin
    this.speedMax = speedmax
    this.gustSpeed = gustspeed
  }
}
